### PR TITLE
Adds a vote to evac shuttle calls.

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Players can now vote to accept/reject an evac shuttle call.
* This is intended to allow interesting rounds to go on for longer; if the majority of the crew is interested in continuing a conflict, it won't be ended because of Command.
* If a vote fails, Command is also not to blame; it is played off as a shuttle from CentCom not being available in the region.
* Also fixes a typo in which `is_authenticated` was spelled `is_autenthicated`.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
Tested, voting for a shuttle works and sends messages to the initiator as intended.

**Comment by Conspiir** [comment]
Feedback thread?

**Comment by KingOfThePing** [comment]
This sounds like an absolute terrible idea. I highly encourage a feedback thread.

**Comment by skull132** [comment]
Doesn't compile. plz fix

File: code/controllers/subsystems/vote.dm [hunk 1]
```diff
@@ -247,20 +255,20 @@ var/datum/controller/subsystem/vote/SSvote
...
 					AddChoice(ROUNDTYPE_STR_MIXED_SECRET, "Mixed Secret")
 			if("crew_transfer")
-				if(check_rights(R_ADMIN|R_MOD, 0))
-					question = "End the shift?"
-					AddChoice("Initiate Crew Transfer")
-					AddChoice("Continue The Round")
-				else
-					if (get_security_level() == "red" || get_security_level() == "delta")
+				if(!check_rights(R_ADMIN|R_MOD, 0))
+					if (get_security_level() == "blue" || get_security_level() == "red" || get_security_level() == "delta")
```
[the remark this answers is not in the document]
Comment (Arrow768) [reply code/controllers/subsystems/vote.dm]: Not an actual question or request for change.

File: code/controllers/subsystems/vote.dm [hunk 2]
```diff
@@ -247,20 +255,20 @@ var/datum/controller/subsystem/vote/SSvote
 				if(ROUNDTYPE_STR_MIXED_SECRET in choices)
 					AddChoice(ROUNDTYPE_STR_MIXED_SECRET, "Mixed Secret")
 			if("crew_transfer")
-				if(check_rights(R_ADMIN|R_MOD, 0))
-					question = "End the shift?"
-					AddChoice("Initiate Crew Transfer")
-					AddChoice("Continue The Round")
-				else
-					if (get_security_level() == "red" || get_security_level() == "delta")
+				if(!check_rights(R_ADMIN|R_MOD, 0))
```
Comment (ParadoxSpace) [inline code/controllers/subsystems/vote.dm]: Let's try this again, this time with feeling.
This is undocumented in the actual body of the PR, and is a weird change that will result in much longer rounds, as well as antags being able to pretty much instantly deny a transfer at the 1:59 mark. Does that satisfy you, Arrow?

Comment (MarinaGryphon) [reply code/controllers/subsystems/vote.dm]: Antags can already do that... people who use the CT mechanics as a ghetto evac vote should be ashamed.

**Comment by Sindorman** [comment]
Please make a feedback thread and post a link here to it as requested by Conspiir

**Comment by skull132** [comment]
Denied. This was downvoted by literally everyone at the head of the server.

The evac shuttle is an IC thing, and is more than likely actually _preferable_ as a means of resolving the round, since it does so at a high point (or generally right after it) of the round. Continuing from there would end with round burn out otherwise lame continuation. While yes, there are issues with rounds being ended a bit _too_ prematurely (disasters mainly) as a result of this, these can perhaps be solved in other ways.

**Pull request closed without merging**